pipeline/backend/common/script_posix.go

```diff
@@ -53,6 +53,8 @@ fi
 unset CI_NETRC_USERNAME
 unset CI_NETRC_PASSWORD
 unset CI_SCRIPT
+mkdir -p $CI_WORKSPACE
```

Comment:
That will work for command steps, not sure what plugins will do ...

Reply:
Also we should extend the git plugin to allow to chown the content ... via setting

Reply:
See my updated OP WRT to the "clone" plugin example.

With the "clone" plugin, the perms are 775 on `$CI_WORKSPACE`. Hence, all subsequent operations should work.